plugins/docker_binary_builder/test/models/binary_builder_test.rb

```diff
@@ -31,9 +32,29 @@
 
     it 'builds image if docker flag is set for project and dockerfile.build exists' do
       File.expects(:exists?).with(File.join(dir, BinaryBuilder::DOCKER_BUILD_FILE)).returns(true)
+      File.expects(:exists?).with(File.join(dir, BinaryBuilder::PRE_BUILD_SCRIPT)).returns(false)
+      executor.expects(:execute!).with(File.join(dir, BinaryBuilder::PRE_BUILD_SCRIPT)).never
+
+      project.update_attributes(deploy_with_docker: true)
+      builder.build
+      output.string.must_equal [
+        "Connecting to Docker host with Api version: 1.19 ...\n",
```

Comment:
this should be all right as the test is stubbing the version in 
https://github.com/zendesk/samson/blob/wyau/update-docker-binary-plugin/plugins/docker_binary_builder/test/models/binary_builder_test.rb#L15